test/unit/specs/components/DecisioningEngine/createContextProvider.spec.js

```diff
@@ -28,7 +41,25 @@ describe("DecisioningEngine:createContextProvider", () => {
 
     expect(contextProvider.getContext({ cool: "beans" })).toEqual({
       cool: "beans",
-      events: {}
+      events: {},
+      timePageLoaded: 1683286686107,
+      datePageLoaded: 5,
+      dayPageLoaded: 5,
+      currentDate: 5,
+      currentTime: 1683286686107,
+      currentDay: 5,
+      scrollPosition: 0,
+      browserDetails: {
+        browserName: "Google Chrome",
+        browserVersion: "112.0"
```

Comment:
browser version should be tested to be `jasmine.any(String)`, since the browser version will change over time and be inconsistent between developer machines